.circleci/config.yml

```diff
@@ -1,60 +1,56 @@
-version: 2
+version: 2.1
+
+orbs:
+  codacy: codacy/base@1.0.0
+  codacy_plugins_test: codacy/plugins-test@0.4.0
+
 jobs:
-  build:
-    machine: true
+  test_patterns:
+    machine:
+      image: ubuntu-1604:201903-01
     working_directory: ~/workdir
     steps:
-      - checkout
-      - run: 
-          name: Clone test project
-          working_directory: ~/
-          command: |
-            (git -C ~/codacy-plugins-test fetch --all && 
-              git -C ~/codacy-plugins-test reset --hard origin/master) || 
-                git clone git://github.com/codacy/codacy-plugins-test.git ~/codacy-plugins-test
-      - restore_cache:
-          key: dependencies-{{ checksum "~/codacy-plugins-test/build.sbt" }}
-      - run: 
-          name: Compile test project
-          working_directory: ~/codacy-plugins-test
-          command: sbt compile
-      - run: 
-          name: Publish tool docker locally
-          working_directory: ~/workdir
-          command: docker build -t ${CIRCLE_PROJECT_REPONAME}:latest .
-      - save_cache:
-          key: dependencies-{{ checksum "~/codacy-plugins-test/build.sbt" }}
-          paths:
-            - "~/.ivy2"
-            - "~/.m2"
-            - "~/.sbt"
-            - "~/codacy-plugins-test/target"
-            - "~/codacy-plugins-test/project/target"
-            - "~/codacy-plugins-test/project/project"
-            - "~/workdir/target"
-            - "~/workdir/project/target"
-            - "~/workdir/project/project"
-      - run: 
-          name: Test json
-          working_directory: ~/codacy-plugins-test
-          command: sbt -Dcodacy.tests.ignore.descriptions=true "runMain codacy.plugins.DockerTest json $CIRCLE_PROJECT_REPONAME:latest"
+      - attach_workspace:
+          at: ~/
       - run:
-          name: Test patterns
-          working_directory: ~/codacy-plugins-test
-          command: sbt -Dcodacy.tests.noremove=true -Dcodacy.tests.threads=8 "runMain codacy.plugins.DockerTest pattern $CIRCLE_PROJECT_REPONAME:latest"
-      - deploy:
-          name: Push application Docker image
           command: |
-            if [ "${CIRCLE_BRANCH}" == "master" ]; then
-              docker login -u $DOCKER_USER -p $DOCKER_PASS
-              docker tag ${CIRCLE_PROJECT_REPONAME} codacy/${CIRCLE_PROJECT_REPONAME}:latest
-              docker tag ${CIRCLE_PROJECT_REPONAME} codacy/${CIRCLE_PROJECT_REPONAME}:1.0.$CIRCLE_BUILD_NUM
-              docker push codacy/${CIRCLE_PROJECT_REPONAME}:latest
-              docker push codacy/${CIRCLE_PROJECT_REPONAME}:1.0.$CIRCLE_BUILD_NUM
-            fi
-
+            docker build -t $CIRCLE_PROJECT_REPONAME:latest .
+            docker run $CIRCLE_PROJECT_REPONAME:latest codacy_prospector_test.py
```

Comment:
Does this file exists? `codacy_prospector_test.py`

Reply:
it doesn't.. I copied this from a different project and didn't notice.
fixed